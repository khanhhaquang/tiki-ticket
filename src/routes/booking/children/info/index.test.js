import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Info from './index';

describe('Test Booking/Info component', () => {
	test('Snapshot info', () => {
		const component = renderer.create(<Info />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	const component = mount(<Info />);
	test('Check info about cinema not empty', () => {
		expect(component.find('.cinema.movie-name')).toBeTruthy();
		expect(component.find('.cinema.movie-detail')).toBeTruthy();
	});

	test('Check info price', () => {
		expect(component.find('.price.total')).not.toBeNaN();
		expect(component.find('.price.currency')).toBeTruthy();
	});
});
