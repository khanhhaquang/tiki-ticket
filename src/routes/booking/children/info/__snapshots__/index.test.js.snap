// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Booking/Info component Snapshot info 1`] = `
<div
  className="sc-bwzfXH cspWGR info"
>
  <div
    className="movie"
  >
    <div
      className="cinema"
    >
      <span
        className="name"
      >
        CGV Hoàng Văn Thụ
      </span>
      <span
        className="time"
      >
        09:10 ~ 11:10 | 08/07/2019
      </span>
    </div>
    <p
      className="price"
    >
      <span
        className="total"
      >
        0
      </span>
      <span
        className="currency"
      >
        đ
      </span>
      <span
        className="info-logo"
      >
        <img
          alt="info"
          src="/assets/images/info.svg"
        />
      </span>
    </p>
  </div>
  <div
    className="actions"
  >
    <button
      className="sc-bdVaJa jYBURw btn"
    >
      Chọn combo
    </button>
    <button
      className="sc-bdVaJa jYBURw btn"
    >
      Thanh toán
    </button>
  </div>
</div>
`;
